spec/spec_helper.rb

```diff
@@ -1,9 +1,6 @@
 # frozen_string_literal: true
 
 require_relative '../lib/testing_appserver'
-require_relative '../shared_example/documents_account_creation'
-require_relative '../shared_example/documents_filter/docx_xlsx_pptx_users_groups_filter'
-require_relative '../shared_example/documents_filter/documents_folder_filter'
-require_relative '../shared_example/documents_filter/documents_img_media_archives_filter'
+require_relative '../shared_example/all_shared_examples'
```

Comment:
I think `spec/spec_helper/shared_examples` will be a better location, we got this location in `testing-documentserver` and it will be better to consistency between projects